lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -769,6 +773,26 @@ def cmd_irb_help
   def cmd_irb(*args)
     defanged?
 
+    expressions = []
+
+    # Parse the command options
+    @@irb_opts.parse(args) do |opt, idx, val|
+      case opt
+        when '-e'
+          expressions << val
```

Comment:
indentation is a little off, I believe we align the when's under the case, tho my editor always does it this way too.
I'll fix it when I land